docker/Dockerfile

```diff
@@ -48,10 +48,9 @@ RUN rm -rf ~/.cache/pip
 # make sure final image is clean
 FROM builder as launcher
 
-# expose ports for flask app - both server and backoffice
+# expose ports for tornado app - both server and backoffice
 EXPOSE 8888 8890
 
-# default commend when entering the container
-ENTRYPOINT ["/bin/bash", "-c"]
-# default command
-CMD ["./docker/start_server_www.sh"]
+# default command when entering the container
+ENTRYPOINT ["python3", "-m", "scripts.run_server", "--config", "${ICUBAM_CONFIG_FILE:-icubam.toml}"]
```

Comment:
I understand the simplification, but sometimes we also need to enter the docker image to run some scripts in maintenance (scripts/wipe_sensitive_info_db.py, scripts/populate_fake_db.py, scripts/get_url_id.py, ...).
I am not a docker expert, but here we will have to set `--entrypoint` to be able to use it, that's it ?

Reply:
so if i remember correctly, you can always enter the docker container via `docker exec -it <container_id> /bin/bash`